src/styles/utils/_mixins.scss

```
@@ -0,0 +1,5 @@
+@mixin transition-hover {
+  cursor: pointer;
+  transition: color 300ms;
+  transition-timing-function: ease;
```

you can merge those properties
```suggestion
  transition: color 300ms ease;
```